package/MDAnalysis/core/topology.py

```diff
@@ -17,13 +17,58 @@
 Topology object --- :mod:`MDAnalysis.core.topology'
 ===================================================================
 
+
+"""
+
+"""
+# second docstring == multiline comment
+
+TODO Notes:
```

Comment:
When do you plan to address this TODO? Wouldn't and issue be better? TODO's have a tendency to stick around forever.